tools/mqppep/mqppep_preproc.xml

```diff
@@ -0,0 +1,531 @@
+               help="[startCol] PERL-compatible regular expression matching column header having first sample intensity"
+               value="^Intensity[^_]">
+            <sanitizer>
+                <valid initial="string.printable">
+                    <remove value="&apos;"/>
+                </valid>
+            </sanitizer>
+        </param>
+        <param name="intervalCol" type="integer" value="1" min="1"
+               label="Interval between the intensity column of samples"
+               help="[intervalCol] E.g., 1 if subsequent column is next sample; 2 if next sample is two columns away, etc."/>
+        <param name="pst_py_selector" type="select"
+            label="Phosphopeptide enrichment type"
+            help="[enrichment] Were samples enriched for pS and pT, or were they enriched for pY instead?"
+            >
+            <option value="st" selected="true">pST</option>
+            <option value="y">pY</option>
+        </param>
+        <param name="collapseFunc" type="select"
+               label="Intensity merge function"
+               help="[collapseFunc] When a peptide is multiply phosphorylated, how should intensities be merged? [default: sum]"
+               >
+            <option value="sum" selected="true">sum</option>
+            <option value="mean">average</option>
+        </param>
+        <param name="localProbCutoff" type="float" value="0.75" min="0" max="1.0"
+               label="Localization Probability Cutoff"
+               help="[localProbCutoff] See help below for an explanation."
+               />
+        <param name="merge_function" type="select" label="intensity merge-function"
+               help="[merge_function] Specifies how intensities for identical phosphosites should be merged">
+            <option value="sum" selected="true">sum</option>
+            <option value="average">average</option>
+        </param>
+        <param name="protein_fasta" type="data" format="fasta" label="UniProtKB/SwissProt FASTA database"
+               help="[protein_fasta] Sequence database; supply the same FASTA file as you supplied to by MaxQuant"
+               />
+        <param name="networkin" type="data" format="tabular" label="NetworKIN file"
+            help="[networkin] NetworKIN file; see help section below"/>
+        <param name="p_sty_motifs" type="data" format="tabular" label="pSTY_Motifs file"
+            help="[p_sty_motifs] pS/pT/pY phosphorylation site motifs; see help section below"/>
+        <param name="psp_kinase_substrate" type="data" format="tabular" label="PSP_Kinase_Substrate_Dataset"
+            help="[psp_kinase_substrate] 'Kinase-substrate dataset'; see help section below"/>
+        <param name="psp_regulatory_sites" type="data" format="tabular" label="PSP_Regulatory_sites"
+            help="[psp_regulatory_sites] 'Regulatory sites'; see help section below"/>
+        <param name="species"
+               type="text"
+               value = "human"
```

Comment:
is that a tax ID, where do users get this string?

Reply:
@bgruening This an optional argument; the commissioner of the tool wanted to be able to filter on species name as they do presently in the functionality of the script that I am trying to wrap.  I will ask what practical value this delivers, i.e., if they can filter the inputs somehow so that this step can be eliminated.